src/linear_rgb.rs

```diff
@@ -7,6 +7,13 @@ use crate::{
     ConversionError, CreationError, Hsl, Rgb, Xyb, Yuv,
 };
 
+/// Contains an RGB image in a linearized RGB color space.
+///
+/// The image is stored as pixels made of three 32-bit floating-point RGB components which are
+/// considered to be in an unspecified linearized color space.
```

Comment:
Currently when converting to linear, we convert to BT709. Though when a user builds this struct, it could be in any set of color primaries. I do wonder what the most "technically correct" thing to do is here. Not something to worry about for this PR, but a thought I had.

Reply:
Maybe include the color primaries in LinearRgb? There could even be a case for some crate that just provides completely correct color space metadata... idk

Reply:
Adding a field here could make sense. I suppose it only truly matters if the conversion from LinearRgb to the new color model depends on specific primaries. With HSL it should in theory work with whatever the LinearRgb is using, but some like XYB may be expecting specific primaries.

Anyway, not something we need to change right at this time.